.pipeline/lib/deploy.js

```diff
@@ -23,19 +23,21 @@ const getParamsByEnv = (env, pr) => {
     SSO_BASE_URL_VALUE: 'https://sso.pathfinder.gov.bc.ca',
     SSO_CLIENT_ID_VALUE: 'devhub-web',
     SSO_REALM_NAME_VALUE: 'devhub',
+    ALGOLIA_INDEX_NAME_SUFFIX: 'prod',
     // for the time being we only have a dev instance, this will change as the api is developed
     SEARCHGATE_API_URL: 'https://searchgate.pathfinder.gov.bc.ca/',
   };
   switch (env) {
     case ENVS.PROD:
       return params;
     case ENVS.TEST:
-      return { ...params, SSO_BASE_URL_VALUE: 'https://sso-test.pathfinder.gov.bc.ca' };
+      return { ...params, SSO_BASE_URL_VALUE: 'https://sso-test.pathfinder.gov.bc.ca', ALGOLIA_INDEX_NAME_SUFFIX: 'test' };
     case ENVS.DEV:
       return {
         ...params,
         SSO_BASE_URL_VALUE: 'https://sso-dev.pathfinder.gov.bc.ca',
         SSO_CLIENT_ID_VALUE: `devhub-web-${pr}`,
+        ALGOLIA_INDEX_NAME_SUFFIX: `-build-${pr}`,
```

Comment:
build in dev?

Reply:
no builds, its just the name of the suffix. this is the same suffix used by the dev image. may as well make them same same